iphone/Maps/Classes/CustomViews/NavigationDashboard/MWMNavigationDashboardManager+Entity.mm

```diff
@@ -56,50 +56,19 @@
   using namespace routing::turns;
   NSString *imageName;
   switch (t)
-   {
-     case PedestrianDirection::TurnRight: imageName = @"simple_right"; break;
-     case PedestrianDirection::TurnLeft: imageName = @"simple_left"; break;
-     case PedestrianDirection::ReachedYourDestination: imageName = @"finish_point"; break;
-     case PedestrianDirection::GoStraight:
-     case PedestrianDirection::Count:
-     case PedestrianDirection::None: imageName = @"straight"; break;
-   }
+  {
```

Comment:
nit: looks like in this file in ObjC `{` should be on the previous line, after the `switch (t)`.